app/views/layouts/_google_analytics.html.erb

```diff
@@ -2,9 +2,10 @@
 <script async src="https://www.googletagmanager.com/gtag/js?id=<%= SITE_CONFIG['google_analytics_script_id'] %>"></script>
 
   <script>
-    dataLayer = [{
-      'signedIn': '<%= user_signed_in?.to_json %>',
+    window.dataLayer = [{
+      'signedIn': '<%= user_signed_in?.to_json %>',
       'hasIPs': '<%= (!!current_user && current_organisation.ips.any?).to_json %>'
+      'superAdmin': '<%= (user_signed_in? && super_admin?).to_json %>'
```

Comment:
I'm wondering how necessary this is as Super Admins will never visit the setup instructions page, they'll never add a location and they'll never add an IP.

They do visit the `/logs` and all other `/logs/something` endpoints, but everything else has `/admin/` preceeding it. So I would say we don't need it.